datadotworld/datadotworld.py

```diff
@@ -51,12 +51,13 @@ class DataDotWorld(object):
         REST API client object
     """
 
-    def __init__(self, config=None):
+    def __init__(self, config=None, **kwargs):
```

Comment:
No need to accept `auth_token` via `kwargs`. If a client wanted to pass the token inline, they could just construct an inline config and pass it in the existing config param here (i.e. `dw = DataDotWorld(config=InlineConfig(token))`)